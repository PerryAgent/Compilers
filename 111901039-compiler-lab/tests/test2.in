x := 5;
for i := 0 to 100
do
    print x + i; 
done;
