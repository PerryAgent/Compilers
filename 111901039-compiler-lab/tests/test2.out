.globl main
main:
li $t1, 5
li $t2, 0
li $t3, 100
Label_No_1:
bgt $t2, $t3, Label_No_2
move $t5 $t1
move $t6 $t2
add $t4, $t5, $t6
move $a0 $t4
li $v0, 1
syscall
addi $t2, $t2, 1
j Label_No_1
Label_No_2:
li $v0, 10
syscall
